policyengine_us/variables/gov/states/or/tax/income/liheap/or_liheap_payout.py

```diff
@@ -0,0 +1,27 @@
+from policyengine_us.model_api import *
+
+
+class liheap_payout(Variable):
+    value_type = float
+    entity = SPMUnit
+    definition_period = YEAR
+    defined_for = StateCode.OR
+    label = "LIHEAP Payout"
+
+    def formula(spm_unit, period, parameters):
+
```

Comment:
add some logic similar to:

electricity_type = tax_unit("or_liheap_electricity_type", period) 

multiplier_based_on_electricity_type = where(utility_type == ELECTRICITY & electricity_type == BOTH , 2, 1)